operations/iscsi_sbps/Managing_selective_node_personalization.md

```diff
@@ -0,0 +1,218 @@
+
+For details, see [After initial CSM 1.7 install or upgrade](#after-initial-csm-17-install-or-upgrade).
+
+### CSM upgrade from 1.6 to 1.7
+
+During an upgrade from CSM 1.6 to CSM 1.7, if the administrator does not wish to enable this feature, no special steps need to be taken. If the administrator does wish to enable this feature, then the following steps need to be followed at the
+beginning of [2. Execute the IUF `management-nodes-rollout` stage](../iuf/workflows/management_rollout.md#2-execute-the-iuf-management-nodes-rollout-stage).
+
+1. Create the `iscsi_worker` HSM group.
+
+   See [Creating the group](#creating-the-group).
+
+1. Perform manual DNS update.
+
+This step is required any time a worker node is changed from being enabled for iSCSI to being disabled for iSCSI.
+In CSM 1.6, all worker nodes were enabled for iSCSI. Therefore, this step will always be required when first enabling this feature on a system that previously was running CSM 1.6.
+
+See [Manual DNS records update](#manual-dns-records-update).
+
+### Adding or removing worker NCN
+
+When [adding or removing a worker NCN](../node_management/Add_Remove_Replace_NCNs/Add_Remove_Replace_NCNs.md), the
+administrator may also wish to create, modify, or delete the `iscsi_worker` HSM group, in order to have the desired
+set of worker NCNs enabled as iSCSI targets. If this is the case, then this should be done **before** adding or
+removing the worker NCN.
+
+For the procedure itself, see [After initial CSM 1.7 install or upgrade](#after-initial-csm-17-install-or-upgrade).
+
+### After initial CSM 1.7 install or upgrade
+
+If this is being done in the context of adding or removing a worker NCN, these steps should be done **before**
+ carrying out that procedure.
+
+During normal system operation, this feature can be enabled or disabled, or the set of worker nodes being enabled
+for iSCSI can be changed. To do this, use the following procedure:
+
+1. Create, modify, or delete the `iscsi_worker` HSM group.
+
+    * If this feature is currently disabled (meaning that all worker nodes are enabled as iSCSI targets), then it can be
+      enabled by creating the HSM group and specifying which worker nodes should be enabled for iSCSI.
+      See [Creating the group](#creating-the-group).
+    * If this feature is currently enabled (meaning that the `iscsi_worker` HSM group exists),
+      then there are two options:
+
+```text
+    * The feature can be disabled (meaning that all worker nodes will be enabled as iSCSI targets) by deleting
+      the group. See [Deleting the group](#deleting-the-group).
+          
+    * The set of worker NCNs being enabled for iSCSI can be modified by changing the membership of the group.
+      This can be done by adding workers to it, removing workers from it, or a combination of both.
+      See [Adding a worker to the group](#adding-a-worker-to-the-group) and [Removing a worker from the group](#removing-a-worker-from-the-group).
+```
```

Comment:
```suggestion
        * The feature can be disabled (meaning that all worker nodes will be enabled as iSCSI targets) by deleting
          the group. See [Deleting the group](#deleting-the-group).
        * The set of worker NCNs being enabled for iSCSI can be modified by changing the membership of the group.
          This can be done by adding workers to it, removing workers from it, or a combination of both.
          See [Adding a worker to the group](#adding-a-worker-to-the-group) and
          [Removing a worker from the group](#removing-a-worker-from-the-group).
```